lib/test_sites/listing.rb

```diff
@@ -104,16 +106,13 @@ def hours_array
     end
 
     def street_address_fallback
-      @address_fallback ||=
```

Comment:
I assume you dropped the memoization here intentionally? I guess it's fine, the call to the StreetAddress parser probably isn't that expensive.

Reply:
My bad, didn't realize that was the intent there—it was late! If we see performance issues, we should restore it, but perhaps at the wrapping call `Listing#street_address` instead.